### ISSUE TITLE
Dozens of Temp files that pile up

### ISSUE THREAD

**Issue by SamuraiThomas** [TwitchRecover/TwitchRecover#48]
I've been wondering for a few days why my SSD has been filling up to the brim.
I opened Treesize to investigate the size of folders on my C: drive so i could see where all that data was being written because i had no idea where all my storage was going. If i open Treesize, i can go into users/name/appdata/local/temp/ and there are dozens of TwitchRecover folders, some of which are 18GB in size and one has 45,000 files inside of it, while the rest mostly have 1000. What is all of this?!
https://imgur.com/a/4QapuyA
![twitchrecover](https://user-images.githubusercontent.com/63924624/107238978-51466e00-6a20-11eb-8551-e0215df201ea.png)

**Comment by SamuraiThomas** [comment]
Just looked further into it, and found that most files are 10 or so second parts of the videos that are probably compiled into the final product. Is there anyway that after the compilation, the TwitchRecover program can automatically delete the temp files?

**Comment by daylamtayari** [comment]
Yes, those are all temporary files that are used to create the final file.
They should all be deleted when you properly exit the program and when I tested it, that would work fine.
Do you force quit the program when you are done with whatever task you are doing?

**Comment by daylamtayari** [comment]
I am going to test edge cases and fix any edge cases that cause temporary files to not be deleted but if you properly exit the program all of the temporary files should be deleted.

**Comment by SamuraiThomas** [comment]
Yes, thanks for the clear up. And i'm not sure what you mean by properly exit the program. I always just use the top right corner X next to the maximise and minimise. Is this the proper way to exit or is there another way? After using the application again, and exiting this way, it again kept a few GB of files in temp.

**Comment by daylamtayari** [comment]
By properly exiting the program I meant answering 'n' for if you want to perform the same operation again and then 'n' again for if you want to perform another answer.
But if these files are not deleted when you just close the program with the x, then that is definitely a bug that needs to be fixed.
